x/ecocredit/server/basket/features/msg_update_basket_fee.feature

```diff
@@ -0,0 +1,117 @@
+Feature: Msg/MsgUpdateBasketFee
+
+  The basket fee can be updated:
+  - when the authority is a governance account address
+  - when the provided basket fee is valid
```

Comment:
We test valid basket fees in basic message validation. The server unit tests only need to test behavior.

```suggestion
```